client/my-sites/guided-transfer/transfer-unavailable-card.jsx

```diff
@@ -33,26 +33,23 @@ class TransferUnavailableCard extends Component {
 			return (
 				<div>
 					<Notice status="is-warning" showDismiss={ false }>
-						{ translate( `It looks like there are some customizations to your site
-						that can't be transferred.` ) }
+						{ translate(
+							"It looks like there are some customizations to your site that can't be transferred."
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **5** times:
`translate( 'It looks like there are some customizations to your site
						that can't be transferred.' )` *ES Score: 10*